docs/legacy/feature-roles.asciidoc

```diff
@@ -34,124 +32,7 @@ In general, there are three types of privileges you'll work with:
-
-IMPORTANT: Setting up {beatname_uc} is an admin-level task that requires extra
-privileges. As a best practice, grant the setup role to administrators only, and
-use a more restrictive role for event publishing.
-
-Administrators who set up {beatname_uc} typically need to load the index template,
-set up ILM aliases, templates, and policies, and register ingest pipelines.
-
-To grant users the required privileges for setup:
-
-. Create a *setup role*, called something like +{beat_default_index_prefix}_setup+, that has
-the following privileges:
-+
-[options="header"]
-|====
-|Type | Privilege | Purpose
-
-|Cluster
-|`manage_ilm`
-|All index lifecycle management operations related to managing policies
-
-|Index
-|`manage` on +{beat_default_index_prefix}-*+ indices
-|All index administration privileges
-|====
-
-. Assign the *setup role*, along with the following built-in roles, to users who
-need to set up {beatname_uc}:
-+
-[options="header"]
-|====
-|Role | Purpose
-
-|`ingest_admin`
-|Set up index templates and ingest pipelines
-|====
-
-NOTE: These instructions assume that you are using the default name for
-{beatname_uc} indices (+{beat_default_index_prefix}-*+).
-If you are using a custom index name, modify the privileges to
-match your index naming pattern.
-
-[float]
-==== Partial setup
-
-In some instances, you may want to create a less-privileged user for a specific setup task.
-
-[float]
-===== Set up index templates
-
-APM Server loads an index template to Elasticsearch at setup time.
-The following privileges are required for this task:
-
-[options="header"]
-|====
-|Type | Privilege | Purpose
-
-|Cluster
-|`manage_index_template`
-|All operations on index templates
-|====
-
-See <<configuration-template>> for more information.
-
-[float]
-===== Set up ILM
-
-Index lifecycle management (ILM) allows you to automate the lifecycle of your APM Server indices as they grow and age.
-To set up and configure ILM, the following privileges are required:
-
-[options="header"]
-|====
-|Type | Privilege | Purpose
-
-|Cluster
-|`manage_index_template`
-|All operations on index templates
-
-|Cluster
-|`manage_ilm`
-|All index lifecycle management operations related to managing policies.
-
-|Index
-|`manage` on +{beat_default_index_prefix}-*+ indices
-|All index administration privileges
-|====
-
-See <<ilm-reference>> for more information.
-
-[float]
-===== Set up ingest pipelines
-
-Ingest pipelines allow you to pre-process documents in Elasticsearch before indexing them.
-The following privileges are required to manage pipelines:
-
-[options="header"]
-|====
-|Type | Privilege | Purpose
-
-|Cluster
-|`manage_pipeline`
-|All operations on ingest pipelines
-|====
-
-See <<pipeline-option-es>> for more information.
-
-////
-***********************************  ***********************************
-***********************************  ***********************************
-////
+// QQ: How are privs for writing events impacted by data stream changes?
```

Comment:
How are privileges for writing events impacted by data stream changes? Are [these docs](https://www.elastic.co/guide/en/apm/server/current/privileges-to-publish-events.html) still accurate?

Reply:
Thanks! Fixed in 7eaea87.